### PR TITLE
chore(DatePicker): date only

### PR DESCRIPTION
**Pull request by jlame**
**What is the problem this PR is trying to solve?**
The time picker was always displayed

**What is the chosen solution to this problem?**
Adding a prop to show/hide it

**Please check if the PR fulfills these requirements**

* [ ] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/src/DateTimePickers/InputDateTimePicker/InputDateTimePicker.component.js [hunk 1]
```diff
@@ -44,11 +47,13 @@ class InputDateTimePicker extends React.Component {
 		onBlur: PropTypes.func,
 		dateFormat: PropTypes.string,
 		useSeconds: PropTypes.bool,
+		useTime: PropTypes.bool,
 	};
 
 	static defaultProps = {
 		dateFormat: 'YYYY-MM-DD',
 		useSeconds: false,
+		useTime: true,
```
Comment (jsomsanith) [inline packages/components/src/DateTimePickers/InputDateTimePicker/InputDateTimePicker.component.js]: We should not have time by default.
So the classical use is only date, and when we want time, we just write `<InputDateTimePicker useTime>`.
(And I think we should rename the component to `InputDatePicker`)

File: packages/components/src/DateTimePickers/InputDateTimePicker/InputDateTimePicker.component.js [hunk 2]
```diff
@@ -63,11 +68,16 @@ class InputDateTimePicker extends React.Component {
 		}
 
 		checkSupportedDateFormat(props.dateFormat);
+		const selectedDate =
+			props.useTime === false
```
Comment (jsomsanith) [inline packages/components/src/DateTimePickers/InputDateTimePicker/InputDateTimePicker.component.js]: if default is `false` no need to test if it's explicitly false, just use the truthyness `props.useTime ? ... : ...`

File: packages/components/src/DateTimePickers/views/DateTimeView/DateTimeView.component.js [hunk 3]
```diff
@@ -89,14 +102,7 @@ class DateTimeView extends React.PureComponent {
...
 				</div>
-				<div className={theme.time}>
-					<TimePicker
-						allowFocus={this.props.allowFocus}
-						value={this.props.selectedTime}
-						onChange={this.props.onSelectTime}
-						useSeconds={this.props.useSeconds}
-					/>
-				</div>
+				{this.getTimePicker()}
```
Comment (jsomsanith) [inline packages/components/src/DateTimePickers/views/DateTimeView/DateTimeView.component.js]: Can you replace that by `this.props.useTime && this.getTimePicker()`. Then in getTimePicker, you don't need the ternary

File: packages/components/src/DateTimePickers/InputDateTimePicker/InputDateTimePicker.component.js (hunk 1 above)
Comment (jlame) [reply packages/components/src/DateTimePickers/InputDateTimePicker/InputDateTimePicker.component.js]: i guess if that component exist its already used somewhere so renaming it my lead to breaking changes ?

Comment (jsomsanith) [reply packages/components/src/DateTimePickers/InputDateTimePicker/InputDateTimePicker.component.js]: No it's not used, as it's still in progress. It's a breaking changes but as it `should` not be used (warning in constructor) and it is not used, it's ok to do this breaking change.
But no pb to do that in another PR

**Pull request merged**